packages/connect/src/api/ethereumSignTypedData.ts

```diff
@@ -91,16 +94,26 @@ export default class EthereumSignTypedData extends AbstractMethod<'ethereumSignT
         }
     }
 
-    async run() {
-        const cmd = this.device.getCommands();
-        const { address_n } = this.params;
+    async initAsync() {
+        if (this.params.network) return;
```

Comment:
Why this `network` condition and missing `chainId` from `getEthereumDefinitions`?